nf_core/pipeline-template/{{cookiecutter.name_noslash}}/.github/workflows/branch.yml

```diff
@@ -13,4 +13,4 @@ jobs:
       # PRs are only ok if coming from an nf-core `dev` branch or a fork `patch` branch
       - name: Check PRs
         run: |
-          { [[ $(git remote get-url origin) == *{{cookiecutter.name_noslash}} ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
+          { [[ $(git remote get-url origin) == *{{cookiecutter.name}} ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
```

Comment:
```suggestion
          { [[ $(git remote get-url origin) == *{{cookiecutter.short_name}} ]] && [[ ${GITHUB_HEAD_REF} = "dev" ]]; } || [[ ${GITHUB_HEAD_REF} == "patch" ]]
```

Reply:
See https://github.com/nf-core/atacseq/pull/70#discussion_r357976331

Reply:
Just some amendments whilst they are fresh @ggabernet @apeltzer

Reply:
@drpatelh I just added the change, but now the tests fail. Why should it be the short_name and not the full pipeline name?

Reply:
Yes, you are right @ggabernet . Sorry, I missed that this test was for merges to `master`. Should really read the documentation...

Reply:
hey no worries, I wan't sure either if that was the correct one :)